content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+    {{< /tip >}}
+
+2. Verify the installed components using the following commands:
+
+    {{< text bash >}}
+    $ code
+    {{< /text >}}
+
+## Deploying an Application
+
+Lets first deploy a sample application composed of four separate microservices used to demonstrate various L7 feature without making it part of the Istio ambient mesh. We can pick from the apps in the samples folder of the istio repository. Execute the following examples from the top of a local Istio repository or istio folder created by downloading the istioctl client as described in istio guides.
+
+{{< text bash >}}
+$ code for bookinfo
+{{< /text >}}
+
+## Deploying a Waypoint Proxy
+
+Let's see how you can Deploy a sample application bookinfo to use the Waypoint proxy
+
+**How to deploy a Waypoint proxy using istioctl**
+TODO
+
+**How to deploy a Waypoint proxy using Helm**
+TODO
+
+## Verify the waypoint proxy is deployed
+
+{{< text bash >}}
+$ code for verification
+{{< /text >}}
+
+This indicates the L7 waypoint proxy is working.  In the next section we look at how to monitor the confuguration and data plane of the L7 waypoint proxy to confirm that traffic is correctly using the L7 waypoint proxy. 
+
+### Verify that the Waypoint proxy is routing traffic to the application
+
+## Configuring the Waypoint Proxy - 
+
+### Overview
+
+This section describes how to configure the waypoint proxy for the Bookinfo application. The Bookinfo application is a sample application that requires a virtual service to route traffic to its different services.
+
+The core functionality of the waypoint L7 traffic management is identical to sidecar mode, hence to add more features refer to this link: https://istio.io/latest/docs/reference/config/networking/virtual-service/
+
+FOR EACH SECTION:
+
+### Creating a Virtual Service
+
+To create a virtual service for the Bookinfo application, you can use the following YAML manifest:
+
+{{< text bash >}}
+$ yaml file
+{{< /text >}}
+
+This virtual service will route all traffic to the `bookinfo-v1` service.
+
+### Deploying the Virtual Service
+
+To deploy the virtual service, you can use the following command:
+
+{{< text bash >}}
+$ command
+{{< /text >}}
+
+### Verifying the Waypoint Proxy Configuration is working
+
+Once the virtual service is deployed, you can verify the waypoint proxy configuration by running the following command:
+
+{{< text bash >}}
+$ command
+{{< /text >}}
+
+This will output the configuration of the waypoint proxy, including the virtual service that is mapped to it.
+
+### Configuring Virtual Services
+
+You can configure both L4 and L7 virtual services for waypoint proxies. If you want to do a Virtual Service with TCP that is effectively a L4 virtual services. This is used to route traffic to services based on port number. L7 virtual services are used to route traffic to services based on more complex criteria, such as HTTP method and path. In a Virtual service you can have only TCP, or only HTTP or Both.
```

Comment:
```suggestion
You can configure both L4 and L7 virtual services for waypoint proxies. If you want to do a Virtual Service with TCP that is effectively a L4 virtual services. This is used to route traffic to services based on port number. L7 virtual services are used to route traffic to services based on more complex criteria, such as HTTP method and path. In a Virtual service you can have only TCP, or only HTTP or Both.
```
A VirtualService defines a set of traffic routing rules to apply when a host is addressed. The functional behavior of VirtualService is identical in Istio ambient mode as in sidecar mode.  Refer to the Istio Traffic Management and VirtualService guides for the exact definition and details. Each routing rule defines matching criteria for traffic of a specific protocol (which currently can be either TCP or HTTP).